Sources/Camille/Configs/Configs+CrossPostService.swift

```diff
@@ -1,7 +1,7 @@
 import Sugar
 
 extension Configs {
-    static let CrossPostService = CrossPostServiceConfig(
+    static let CrossPost = CrossPostServiceConfig(
```

Comment:
If everyone is cool with this style change being in another PR lemme know and I will merge this 👍